omega/omega-context/src/main/java/org/apache/servicecomb/saga/omega/context/OmegaContext.java

```diff
@@ -27,6 +30,12 @@
   private final ThreadLocal<String> globalTxId = new InheritableThreadLocal<>();
   private final ThreadLocal<String> localTxId = new InheritableThreadLocal<>();
   private final IdGenerator<String> idGenerator;
+  private final ThreadLocal<Map<String, Object[]>> parameters = new InheritableThreadLocal<Map<String, Object[]>>() {
```

Comment:
The parameters could eat up the memory, and we need to remove it after the invocation is successed.
Don't worry, I will add a quick fix for it.